src/screens/block-details.tsx

```diff
@@ -155,11 +155,12 @@ export default function BlockDetails() {
                             </LabelledContent>
                             <LabelledContent label="To">
                               <Text wrap={false} size="12px">
-                                {transaction.to &&
-                                  truncate(transaction.to, {
-                                    start: 6,
-                                    end: 4,
-                                  })}
+                                {truncate(
```

Comment:
Happy to just leave the `to` blank here. Some transactions may not have a `to` address, and that doesn't mean that it's default to the burn addr.